app/admin/organisations.rb

```diff
@@ -17,7 +17,7 @@
     selectable_column
     id_column
     column :name
-    column :org_type
+    column "org_type", :providertype
```

Comment:
Would suggest "Org type" or "Organisation type" so it's humanized, consistent with other table labels.